test/acceptance/docker/deploy.sh

```diff
@@ -1,6 +1,8 @@
 #!/bin/bash
 
 git clone https://github.com/CartoDB/Windshaft-cartodb.git ;
+git checkout master
```

Comment:
Are these really needed? A clean `clone` just was performed

Reply:
I'm not sure. 🤷‍♂️ 

What happens when you already have a local copy?

Reply:
(from docs) It seems that clones checks out currently active branch, by default
https://git-scm.com/docs/git-clone

Reply:
I see the intention now, but, the clone will create a new directory and the checkout and pull will be performed in the parent directory, right?

Also, do we want to enforce the checkout+pull? Isn't it useful to be able to work with a local Windshaft?

Reply:
Actually, I just tested and it doesn't work as I was expecting. :\

If I check out an older commit than master:HEAD, expecting that the deploy script would check out master:HEAD again, but it doesn't.

Reply:
I think it's not a problem for CI. I'm gonna remove this and push a new docker image.

I was trying to fix the situation with outdated Windshaft-cartodb repositories in local/dev environments.